client/whispers/whisper.jsx

```diff
@@ -90,11 +90,11 @@ class Whisper extends React.Component {
 function isClosingCurrentWhisperSession(oldProps, newProps) {
```

Comment:
Not something you changed here, but these sorts of functions seems to be a prime refactoring target, to me. We duplicate them for basically every subject area with slight differences, and have to check them in a number of spots. Seems like we could make a wrapper component that would do this check (and receive the data from the store), leaving a pure component inside that receives that data on props and could not deal with that stuff?

Dunno, just something to think about for the future